src/components/engagement_data_cards/user_card/user_card.tsx

```diff
@@ -30,6 +30,12 @@ export function UserCard({
     requestClose();
     onClear();
   };
+
+  function onCancel() {
+    requestClose();
+    window.location.reload();
```

Comment:
do we need to reload the window here? what is this accomplishing?

Reply:
UI is using the same user object that has been manipulated in the modal. By refreshing here it loads new users and not read them from memory. 
Ideally, we should refactor the whole class to not manipulate the object itself, I tried to do that but it got really messy, as we're having multiple rows and that added more complication. The easiest solution is reloading the page after user clicks cancel.